src/App.tsx

```diff
@@ -143,15 +147,21 @@ export default function App() {
           <Text>Tip: {state.tip.toFixed(2)}</Text>
         </>
         <Text>Subtotal: {state.total}</Text>
+        <Button
```

Comment:
I didn't write a test for this component since I wasn't sure if the test should be written when we actually integrate the API? open to what others think about this

Reply:
Hm.. makes sense. Is there even a way to check what console.log it is? lol. We could have another diff that displays the text too if it's not too much work? Otherwise, I'm fine without test as well. Can you include the full payload of this state in your PR description? Just want to double check how items look like.

Reply:
added like a 1-liner that tests that the submit button appears in the app. also had to add another rule because it was complaining how a package should be listed under dep and not devDep. 

also updated the pr description!